imod/mf6/hfb.py

```diff
@@ -472,13 +494,20 @@ def render(self, directory, pkgname, globaltimes, binary):
             package, first convert to a Modflow6 package by calling pkg.to_mf6_pkg()"""
         )
 
+    def to_netcdf(self, *args, **kwargs):
+        new = deepcopy(self)
+        # kwargs.update({"encoding": self._netcdf_encoding()})
+        new.dataset["geometry"] = new.line_data.to_json()
+        new.dataset.to_netcdf(*args, **kwargs)
+
     def _netcdf_encoding(self):
         return {"geometry": {"dtype": "str"}}
 
     @classmethod
     def from_file(cls, path, **kwargs):
         instance = super().from_file(path, **kwargs)
-        instance.dataset["geometry"] = shapely.wkt.loads(instance.dataset["geometry"])
+        geometry = json.loads(instance.dataset["geometry"].values.item())
```

Comment:
Nice! This might actually fix an issue I ran in with dumping HFB files to netcdf and importing them back again, for which I forgot to post an issue I see... I couldn't quickly reproduce the error unfortunately

Reply:
Give it a shot. This way of saving it should be really robust